Plugins/Flow.Launcher.Plugin.Calculator/Main.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
This intentional?

Reply:
Lol yeah, otherwise the md file changes won't trigger a build. This change should be the same.